pkg/trebuchet/server/options/trebuchet_options.go

```diff
@@ -0,0 +1,20 @@
+package options
+
+import (
+	"github.com/atlassian/ctrl"
+	"github.com/atlassian/ctrl/options"
+)
+
+type TrebuchetOptions struct {
+	LoggingOptions options.LoggerOptions
```

Comment:
TODO: these options are not used anywhere. You can leave a TODO comment for now if you prefer to add this code later.

Reply:
Would it be a bad idea to leave this here, since it's likely that we'll need logging features in the future?

Reply:
@qunayu not sure what you mean. The logging options are definitely useful, they are just not used at the moment, so we need to make sure we don't forget to use them :)

Reply:
Ahh, I see. Thanks!